app/views/alchemy/admin/elements/_element_toolbar.html.erb

```diff
@@ -1,4 +1,4 @@
-<% remarkable_type = element.class.name.demodulize.underscore.pluralize %>
+<% remarkable_type = "elements" %>
```

Comment:
It looks like the local variable isn't necessary here any longer, right? We can just use the string in line 7 and 17...